api/analyzers/java/analyzer.py

```
@@ -127,3 +127,19 @@ def resolve_symbol(self, files: dict[Path, File], lsp: SyncLanguageServer, file_
             return self.resolve_method(files, lsp, file_path, path, symbol)
         else:
             raise ValueError(f"Unknown key {key}")
+
+    def add_file_imports(self, file: File) -> None:
```

Ruff (0.15.5)

[error] `File` may be undefined, or defined from star imports

(F405)